tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+requires =
+    tox-conda
+    tox-run-command
+minversion = 3.15
+envlist=py37,py38,py39,py310
+
+[testenv]
+description = invoke pytest to run automated tests
+deps =
+    pytest
+    pytest-cov
+    filelock
+    requests
+    imagehash
+conda_env = ci/requirements/dev.yml
+commands =
+    pytest --cov tephi --cov-report term-missing {posargs}
+
+[testenv:lint]
```

Comment:
Can we ditch Stickler once this is merged?

Reply:
No idea. what is stickler?

Reply:
> what is stickler?

Old enemy! 🤠 

Stickler is a linting bot.

Reply:
you might need flake8 in here if you want to ditch stickler?
https://github.com/SciTools/tephi/blob/master/.stickler.yml

Reply:
Actually I just noticed pre-commit as already setup for Tephi. Please could we therefore remove the lint job and replace with [pre-commit.ci](https://pre-commit.ci/), like what has been [proposed for Iris](https://github.com/SciTools/iris/pull/4503#issuecomment-1013227400)?

Reply:
Sure. To be clear though, this lint job doesn't run in the test ci. The test ci only runs `[testenv]` section of tox. To run the `lint` you need to explicitly run that section: `tox -e lint`. So this is more of a local dev tool than anything else

Reply:
>So this is more of a local dev tool than anything else

What are your thoughts on just encouraging exclusive use of pre-commit instead?